payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -221,7 +225,7 @@ internal class DefaultCardNumberController(
         } else if (accountRangeService.accountRange != null) {
             TextFieldIcon.Trailing(accountRangeService.accountRange!!.brand.icon, isTintable = false)
         } else {
-            val cardBrands = CardBrand.getCardBrands(number)
+            val cardBrands = CardBrand.getCardBrands(number).filter { cardBrandFilter.isAccepted(it) }
```

Comment:
Are there any tests for this change?

Reply:
There wasn't, but this is surprisingly great/easy to test! Added!